src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Extras/AppShell/BitAppShellDemo.razor.scss

```
@@ -0,0 +1,92 @@
+    flex-wrap: wrap;
+}
+
+.readout {
+    gap: 0.25rem;
+    display: flex;
+    flex-wrap: wrap;
+    column-gap: 1.5rem;
+    color: var(--bit-clr-fg-sec);
+}
+
+::deep .consumer {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)